iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalTableOperations.java

```diff
@@ -63,7 +63,7 @@ public class OpenHouseInternalTableOperations extends BaseMetastoreTableOperatio
   private static final Gson GSON = new Gson();
 
   private static final Cache<String, Integer> CACHE =
-      CacheBuilder.newBuilder().expireAfterWrite(1, TimeUnit.MINUTES).maximumSize(100).build();
+      CacheBuilder.newBuilder().expireAfterWrite(5, TimeUnit.MINUTES).maximumSize(1000).build();
```

Comment:
Is this the estimated size based on the request generated on HTS (with retry) for each tables request?

Reply:
no, this is coming from the volume of the total put request. see the PR description on where we land in the size number. 
TTL is purely heuristic.

Reply:
Sounds good.